Submission

vasilisa-iatckova opened the pull request:
can't see test results in github / on git but they pass when run locally via terminal

axiezai commented:
I turned github actions on, you never turned it on before i guess :) I will review tomorrow though, if there are recommended changes, you can make them through the pull review and the tests should run through github actions.

vasilisa-iatckova commented:
Thanks!

On Mon, Mar 29, 2021 at 10:13 PM Xihe Xie ***@***.***> wrote:

> I turned github actions on, you never turned it on before i guess :) I
> will review tomorrow though, if there are recommended changes, you can make
> them through the pull review and the tests should run through github
> actions.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vasilisa-iatckova/wcm_datasci_basics_06images/pull/1#issuecomment-809851877>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQAWD7J7P3N5RBHVYKZIMA3TGEXUNANCNFSM42BAEYZA>
> .
>

axiezai commented:
I added a pycache gitignore and deleted those files cuz 
 - 1 - they are annoying for code reviews 
 - 2 - people should really add them to global gitignore so they never show up in any python projects on github
 -  3 - now your github actions run

Other than that, wish you used your `df_normalized` more often than `df_aligned` for plotting... or what else is the point of creating that thing? Especially for the 2nd part, the last heatmap is 👎🏼  -2% 

Final grade 98%.

BTW TY for improving the tests and the find middle function. Merge if you want. Leaving it unmerged/open for you to check my comments.

vasilisa-iatckova commented:
Had the same thoughts on normalized but was running on fumes at that point
so lazily ditched that part.

In general, you’re a very generous instructor and grader. I’ve never gotten
this much individual attention in a coding-related course before. Thank you
so much for teaching!

On Tue, Mar 30, 2021 at 5:55 PM Xihe Xie ***@***.***> wrote:

> I added a pycache gitignore and deleted those files cuz
>
>    - 1 - they are annoying for code reviews
>    - 2 - people should really add them to global gitignore so they never
>    show up in any python projects on github
>    - 3 - now your github actions run
>
> Other than that, wish you used your df_normalized more often than
> df_aligned for plotting... or what else is the point of creating that
> thing? Especially for the 2nd part, the last heatmap is 👎🏼 -2%
>
> Final grade 98%.
>
> BTW TY for improving the tests and the find middle function.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vasilisa-iatckova/wcm_datasci_basics_06images/pull/1#issuecomment-810603002>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQAWD7MZYVPT7YMXPUTKARTTGJCDVANCNFSM42BAEYZA>
> .
>
